docs/source/gbd2019_models/risk_effects/x_factor/index.rst

```diff
@@ -39,92 +39,155 @@ Wasting X-Factor
 
+According to [Stobaugh-et-al-2019]_, an unpublished longitudinal study in Ethiopia (Tsineal et al. 2015/Jimma University) reported the probability of experiencing a new episode of SAM or MAM was 26% for 6 months and 7.5% for 12 months. Tsineal et al. 2015 also reportedly found a relapse rate of 15% for SAM and 44% for MAM over 12 months. Further, this study was the only one identified by [Stobaugh-et-al-2019]_ that included a true control group of non-malnourished counterparts: Tsineal et al. 2015 reportedly followed children after SAM treatement as well as non-wasted matched community controls for one year. They reportedly found that 15% of the SAM-treated group developd SAM in the year of follow-up while only 1.2% of control children did in the same period for respective incidence rates of 1.27 and 0.09 per 100 person months.
 
-.. todo::
+Since the time of this review, two additional studies were identified that compared wasting episode relapse rates betwen children who were recently treated for acute malnutrition and children who were not. 
 
-  - A more thorough literature review and support for use of this proxy should be done to strengthen our argument.
-  - We have decided not to model the effect of x-factor on stunting for now. We will look more thoroughly on its effect on stunting and whether to model a direct effect of wasting on stunting as we do more research.
-  - may be helpful to put this in the concept model diagram to keep track of how different factors affect incidence rates
+[Abitew-et-al-2020]_ conducted a cross-sectional study in South Gondar
+Zone, Amhara Region, Ethiopia of children discharged from CMAM programs and aged-matched children who were never treated for acute malnutrition. The study found that 34.2% of children discharged from CMAM programs were wasted compared to 26.7% of the control group. More specifically, 22.3% and 12.0% of the children discharged from CMAM were classified as having MAM and SAM, respectively, compared to 17.2% and 9.5% of the control population. Notably, the difference in wasting burden between these study populations found by [Abitew-et-al-2020]_ is of lower magnitude than other data sources that investigated the same measure. This may be because of the cross-sectional rather than longitudinal study design which measures prevalence rather than cumulative incidence. Further, the inclusion criteria for control subjects was based on whether they had previously received CMAM treatment without consideration of CMAM reatment need. Additionally, the prevalence of SAM in this population is quite high, indicating that there is a high background risk of acute malnutrition in the population that may not be generalizable to other populations with lower SAM prevalence.
```

Comment:
What does CMAM stand for (what is a CMAM program)?

Also, there's a typo: "without consideration of CMAM reatment need" (reatment -> treatment)

Reply:
Oh, it looks like you define community-based management of acute malnutrition (CMAM) below, but you should probably add it here as well since this is the first mention.

Reply:
Updated! Thanks